ui.R

```diff
@@ -64,9 +64,10 @@ shinyUI(fluidPage(
       fluidRow(
         column(6, numericInput(
           "wage1_hourly", "Hourly wage ($)",
-          value = DEFAULT_HOURLY_WAGE, step = .1
+          value = DEFAULT_HOURLY_WAGE, step = .1,
+          min = 15, max = 100

```

Comment:
I'm unsure whether the max and min on the input tickers are necessary when inputs outside this range are valid and can be entered by the user. In addition, minimum wage in TY14 and TY15 were 14.25 and 14.75 respectively.